extensions/pyRevitTools.extension/checks/modelchecker_check.py

```diff
@@ -438,7 +438,9 @@ def checkModel(doc, output):
     # it is something with schedules on more sheets maybe...
```

Comment:
the way the schedule not on sheets collects data seems bloated and not working properly.
note for later, handle it the same way the **Views - Query view sheet placement** tool does